servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H1SpecExceptions.java

```diff
@@ -37,21 +40,64 @@ public boolean allowPrematureClosureBeforePayloadBody() {
         return allowPrematureClosureBeforePayloadBody;
     }
 
+    /**
+     * Allow {@code LF} without a proceeding {@code CR} as described in
+     * <a href="https://tools.ietf.org/html/rfc7230#section-3.5">HTTP/1.x Message Parsing Robustness</a>:
+     * <pre>
+     *   Although the line terminator for the start-line and header fields is
+     *    the sequence CRLF, a recipient MAY recognize a single LF as a line
+     *    terminator and ignore any preceding CR.
+     * </pre>
+     * @return {@code true} to allow {@code LF} without a proceeding {@code CR}.
+     */
+    public boolean allowLFWithoutCR() {
+        return allowLFWithoutCR;
+    }
+
     /**
      * Builder for {@link H1SpecExceptions}.
      */
     public static final class Builder {
 
         private boolean allowPrematureClosureBeforePayloadBody;
+        private boolean allowLFWithoutCR;
 
         /**
          * Allows interpreting connection closures as the end of HTTP/1.1 messages if the receiver did not receive any
          * part of the payload body before the connection closure.
-         *
+         * @deprecated Use {@link #prematureClosureBeforePayloadBody(boolean)}.
```

Comment:
@idelpivnitskiy - this is inline with previous discussion we've had regarding builders allowing state to be controlled externally and re-usable.